examples/kuesa/tonemapping/qml/SceneEntity.qml

```diff
@@ -0,0 +1,174 @@
+                clearColor: Qt.rgba(0.1, 0.1, 0.1, 1.0)
+            }
+        },
+        InputSettings { },
+        EnvironmentLight {
+            irradiance: TextureLoader {
+                source: _assetsPrefix + "pink_sunrise" + envMapFormat + "_irradiance" + ((!root3D.es2) ? ".dds" : "_es2.dds")
+
+                minificationFilter: Texture.LinearMipMapLinear
+                magnificationFilter: Texture.Linear
+                wrapMode {
+                    x: WrapMode.ClampToEdge
+                    y: WrapMode.ClampToEdge
+                }
+                generateMipMaps: false
+            }
+            specular: TextureLoader {
+                source: _assetsPrefix + "pink_sunrise" + envMapFormat + "_specular" + ((!root3D.es2) ? ".dds" : "_es2.dds")
+
+                minificationFilter: Texture.LinearMipMapLinear
+                magnificationFilter: Texture.Linear
+                wrapMode {
+                    x: WrapMode.ClampToEdge
+                    y: WrapMode.ClampToEdge
+                }
+                generateMipMaps: false
+            }
+        }
+    ]
+
+    Entity {
+        Transform {
+            id: transform
+            property real angle: 0.0
+
+            readonly property real radianAngle: angle * 2 * Math.PI / 360
+            readonly property real radius: 10
+            readonly property real x: radius * Math.cos(radianAngle)
+            readonly property real y: 0
+            readonly property real z: radius * Math.sin(radianAngle)
+
+            scale: 10
+            translation: Qt.vector3d(x, y, z)
+
+            NumberAnimation on angle {
+                loops: Animation.Infinite
+                duration: 10000
+                from: 0
+                to: 360
+                paused: root3D.lightRotating
+            }
+        }
+
+        PointLight {
+            id: pointLight
+            color: "white"
+            intensity: 0.2
+        }
+        components: [ pointLight, transform ]
+    }
+
+    Camera {
+        id: fallbackCamera
+        position: Qt.vector3d(0.0, 0.0, 7.0)
+        upVector: Qt.vector3d(0.0, 1.0, 0.0)
+        exposure: root3D.exposure
+        viewCenter: Qt.vector3d(0.0, 0.0, 0.0)
+    }
+
+    OrbitCameraController {
+        id: controller
+        camera: frameGraph.camera
+        linearSpeed: 5
+        lookSpeed: 180
+    }
+
+    Kuesa.AnimationPlayer {
+        id: helmetSpin
+        sceneEntity: root3D
+        clip: "damagedHelmet_Action"
+        loops: Kuesa.AnimationPlayer.Infinite
+        running: root3D.rotating
+    }
+
+    Kuesa.Asset {
+        id: cameraAsset
+        name: root3D.cameras.names.length ? root3D.cameras.names[0] : ""
+        collection: root3D.cameras
+    }
+
+    Kuesa.Asset {
+        id: materialsAsset
+        name: "Material_MR"
+        collection: root3D.materials
+        onNodeChanged: {
+            console.log("material", node)
+            if(node) {
+                if(root3D.tone) {
```

Comment:
you should be able to control this from the UI. Currently it's off by default and the user can't turn it on... what's the point?

Reply:
Done.